js/src/views/Signer/components/TransactionMainDetails/TransactionMainDetails.css

```diff
@@ -19,6 +19,7 @@
 
 .transaction {
   flex: 1;
+  overflow: auto;
```

Comment:
This is to limit the container width. `word-break: break-all` is actually already on the Transaction data container.